online_meetings/report.py

```diff
@@ -36,27 +37,50 @@
 DEFAULT_SLEEP_TIME = ENV.get('DEFAULT_SLEEP_TIME', 10)
 
 
-# Functions
-def get_total_page_count(url: str, headers: Dict[str, Union[str, int]] = None, params: Dict[str, Union[str, int]] = None):
-    if params is None:
-        params = {}
-    if headers is None:
-        headers = {}
+def get_request_retry(url: str, headers: Dict[str, Union[str, int]],
+                      params: Dict[str, Union[str, int]]) -> requests.Response:
+
+    response = requests.request("GET", url, headers=headers, params=params)
```

Comment:
I'm trying to run this, but it seems like the `url` is only the URL ending. Am I missing something?

Reply:
Sorry, zoom base url is not set in the config.